Not compatible with Dokka 0.10.0

[Dokka 0.10.0](https://github.com/Kotlin/dokka/releases/tag/0.10.0) merges the android plugin into the main plugin, so this plugin fails when it tries to apply the `dokka-android` plugin.

Error message looks like this:
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':my_project'.
> Failed to apply plugin [id 'org.jetbrains.dokka-android']
   > Cannot add task 'dokka' as a task with that name already exists.
```

--- a/src/main/groovy/com/vanniktech/maven/publish/MavenPublishPlugin.groovy
+++ b/src/main/groovy/com/vanniktech/maven/publish/MavenPublishPlugin.groovy
@@ -48,7 +48,7 @@ class MavenPublishPlugin extends BaseMavenPublishPlugin {
 
     if (plugins.hasPlugin('kotlin-android')) {
       def dokkaOutput = "${project.docsDir}/dokka"
-      project.plugins.apply('org.jetbrains.dokka-android')
+      project.plugins.apply('org.jetbrains.dokka')
       project.dokka {
         outputFormat 'html'
         outputDirectory dokkaOutput

